python/url-shortener/src/main.py

```
@@ -0,0 +1,60 @@
+        short_id = nanoid.generate(size=7)
+
+        try:
+            database.create_document(
+                database_id,
+                collection_id,
+                short_id,
+                {'original_url': original_url}
+            )
+            short_url = f"{context.req.scheme}://{context.req.host}/v1/databases/{database_id}/collections/{collection_id}/documents/{short_id}"
+            return context.res.json({'short_url': short_url})
+        except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)